src/pages/listpage/CardListPage.jsx

```diff
@@ -0,0 +1,78 @@
+import { RecipientsAPI } from 'data/CallAPI';
+import { useState, useEffect } from 'react';
+import CardList from 'components/card/CardList';
+import TextDropdownField from 'components/textfield/TextDropdownField';
+import TextInputField from 'components/textfield/TextInputField';
+import EmojiToggle from 'components/EmojiToggle';
+import "pages/listpage/CardListPage.scss";
+
+function HotSort(recipient) {
+  if (!recipient || !Array.isArray(recipient.results)) {
+    //레시피언트 없는지와 레시피언트리절트가 배열인지 확인
+    return [];
+  }
+  return [...recipient.results].sort((a,b) => {
+    if(b.messageCount === a.messageCount) {
+      return b.reactionCount - a.reactionCount;
+    }
+    return b.messageCount - a.messageCount;
+  });
+}
```

Comment:
```suggestion
return [...recipient.results].sort((a, b) => b.messageCount - a.messageCount || b.reactionCount - a.reactionCount);
```
요렇게 코드 줄일 수 있을거 같아용 ㅎㅎ
같으면 0이라서 false가 되기 때문에 뒤에 조건을 보게 되니까용!